schedule.json

```diff
@@ -137,5 +137,17 @@
     "maintenance": "2026-10-20",
     "end": "2028-04-30",
     "codename": ""
+  },
+  "v25": {
+    "start": "2025-10-15",
+    "maintenance": "2026-04-01",
+    "end": "2026-06-01"
+  },
+  "v26": {
+    "start": "2026-05-06",
```

Comment:
We usually release the even-numbered ones in April (this year's was delayed into May to accommodate the issues with updating V8).
https://github.com/nodejs/Release/blob/cf57a9739438f7a46d8f49b18aa8c2de3f3ca756/README.md?plain=1#L105-L107